### ISSUE TITLE
Installation Issue: throwing "error 71"

### ISSUE THREAD

**Issue by apkawel** [ValiValpas/taskopen#114]
- Thank you for your work.
- Thank you for your patience and help with this installation issue.

- I am running macOS Mojave 10.14 (18A389) – most-recent beta as of 2018-09-22.
- My perl5: revision 5 version 18 subversion 2.

- I run the installation commands and get the following:

```
$ make PREFIX=/usr
gzip -c doc/man/taskopen.1 > doc/man/taskopen.1.gz
gzip -c doc/man/taskopenrc.5 > doc/man/taskopenrc.5.gz
sed s',#PATH_EXT=.*,&\nPATH_EXT=/usr/share/taskopen/scripts,' taskopen > taskopen.pl

$ make PREFIX=/usr install
gzip -c doc/man/taskopen.1 > doc/man/taskopen.1.gz
gzip -c doc/man/taskopenrc.5 > doc/man/taskopenrc.5.gz
sed s',#PATH_EXT=.*,&\nPATH_EXT=/usr/share/taskopen/scripts,' taskopen > taskopen.pl
mkdir -p //usr/bin
install -m 0755 taskopen.pl //usr/bin/taskopen
install: //usr/bin/taskopen: Operation not permitted
make: *** [install] Error 71
```

- I tried using the solution in Issue #113 but it doesn't seem to make a difference.
- Any pointers would be greatly appreciated; I'm exciting to get taskopen working on my system!

**Comment by apkawel** [comment]
As a follow-up, I tried:

```
$ sudo make PREFIX=/usr/local/
```
and
```
$ sudo make PREFIX=/usr/local/ install
```
and it seemed to install properly.

But when I tried using taskopen, this:
```
$ taskopen 259
-bash: /usr/local/bin/taskopen: /usr/local/bin/perl: bad interpreter: No such file or directory
```

**Comment by Smingvin** [comment]
I'm having the same issue. 

MacOS Sierra 10.12.4

**Comment by ValiValpas** [comment]
@apkawel sorry for the late reply, I must have missed this issue

Taskopen installed correctly but it cannot find the perl binary. It appears you changed the perl path into `/usr/local/bin/perl` as mentioned in #113. Can you please run `which perl` to check where your perl binary is installed?

@Smingvin do you mean you have the same issue with installing taskopen or the same issue executing taskopen?

**Comment by apkawel** [comment]
/usr/bin/perl

**Comment by Smingvin** [comment]
@ValiValpas 

Both issues.

First I got the same error when installing. Then I installed to usr/local like apkawel. When I tried to run it, I get the same error as apkawel. Also, my pearl installation is at "/usr/bin/perl" just as apkawel. 

I redownloaded and tried to reinstall, but when I run make PREFIX=/usr I get the following error:

```
gzip -c doc/man/taskopen.1 > doc/man/taskopen.1.gz
gzip -c doc/man/taskopenrc.5 > doc/man/taskopenrc.5.gz
sed s',#PATH_EXT=.*,&\nPATH_EXT=/usr/share/taskopen/scripts,' taskopen > taskopen.pl
sed -i s',/usr/bin/perl,/usr/local/bin/perl,' taskopen.pl
sed: 1: "taskopen.pl": undefined label 'askopen.pl'
make: *** [taskopen.pl] Error 1
```

Thanks so much for helping out!

**Comment by Smingvin** [comment]
I fixed it! 

I added '.bak' next to -i in that sed line you added in the last commit, like so:

`	sed -i '.bak' s',/usr/bin/perl,$(PERLPATH),' taskopen.pl`

(as [instructed here](https://www.mkyong.com/mac/sed-command-hits-undefined-label-error-on-mac-os-x/.)

Then I tried installing it using the default instructions and it didn't work.

But then I tried @apkawel's 

```
$ sudo make PREFIX=/usr/local/
$ sudo make PREFIX=/usr/local/ install
```
And now it works.
